test/e2e/tests/simple-send.spec.js

```diff
@@ -14,14 +19,28 @@ describe('Simple send', function () {
         ganacheOptions: defaultGanacheOptions,
         title: this.test.fullTitle(),
       },
-      async ({ driver, ganacheServer }) => {
-        await logInWithBalanceValidation(driver, ganacheServer);
+      async ({ driver }) => {
+        // Process
+        await unlockWallet(driver, WALLET_PASSWORD);
 
-        await sendTransaction(
-          driver,
+        // Page Objects and Components
+        const confirmTxPage = new ConfirmTxPage(driver);
+        const homePage = new HomePage(driver);
+        const sendAmountPage = new SendAmountPage(driver);
+        const sendToPage = new SendToPage(driver);
+
+        // Actions and Assertions
+        await homePage.startSendFlow();
+        await sendToPage.addRecipient(
           '0x985c30949c92df7a0bd42e0f3e3d539ece98db24',
-          '1',
         );
+        await sendAmountPage.addAmount('1');
+        await sendAmountPage.goToNextScreen();
+        await confirmTxPage.confirmTx();
+        await homePage.goToActivityList();
+
+        const confirmedTx = await homePage.isConfirmedTxInActivity();
+        assert.equal(confirmedTx, true, 'Confirmed tx is not found');
       },
     );
   });
```

Comment:
As we're migrating to TypeScript, it might be an ideal time to also adopt it when we implementing the POM. I suggest we consider writing our code in TypeScript during this process.